portfolio/src/main/webapp/contact.html

```diff
@@ -49,6 +49,11 @@ <h2>COMMENT BELOW</h2>
                     <p>COMMENT</p>
                     <input name="text-input" required>
                     <br/><br/>
+                    <p>IMAGE UPLOAD</p>
+                    <input type="file" name="image">
+                    <button id="fileButton" onclick="getFileURL();return false;">UPLOAD</button>
```

Comment:
Note sure what `getFileURL()`, but the name doesn't sound right here. Maybe, `uploadFile()`?